Public/Src/Engine/UnitTests/Engine/SharedOpaqueEngineTests.cs

```diff
@@ -82,6 +82,35 @@ public void OutputsUnderSharedOpaqueAreSelectivelyScrubbed()
             Assert.True(File.Exists(Path.Combine(objDir, outOfBuildFile)));
         }
 
+        [Fact]

```

Comment:
>[Fact] [](start = 8, length = 6)

Why don't you make it as [Theory] where the inline value is true for disabling the scrubbing and false for enabling the scrubbing.